File: dotnet/Src/ButterflyNet.Parser/Scopes/StrongScope.cs
namespace ButterflyNet.Parser.Scopes {
	public class StrongScope : InlineScope {
		public override void Open(ButterflyAnalyzer analyzer) {
			analyzer.OpenStrongText();
		}

		public override void Close(ButterflyAnalyzer analyzer) {
			analyzer.CloseStrongText();
		}
	}
}

File: src/js/src/scopes/StrongScope.js
function StrongScope() {
	this.type = ScopeTypeCache.strong;
	this.level = ScopeLevel.inline;
	this.canNestText = true;
	
	this.open = function(analyzer) {
		analyzer.openStrongText();
	};
	this.close = function(analyzer) {
		analyzer.closeStrongText();
	};
}

extend(Scope, StrongScope);